snippets/dart_2_snippets.md

```diff
@@ -0,0 +1,53 @@
+## Available Dart 1 and Dart 2 Backwards Compatible Code Snippets
```

Comment:
What do we think about renaming this file `README.md`? That way users are greeted with a rendered markdown file when they nav to this directory, rather than just a list of files.